config/network/errors.go

```diff
@@ -0,0 +1,30 @@
+package network
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/onflow/flow-go/network/p2p"
+)
+
+// ErrInvalidLimitConfig indicates the validation limit is < 0.
+type ErrInvalidLimitConfig struct {
+	// controlMsg the control message type.
+	controlMsg p2p.ControlMessageType
+	err        error
+}
+
+func (e ErrInvalidLimitConfig) Error() string {
+	return fmt.Errorf("invalid rpc control message %s validation limit configuration: %w", e.controlMsg, e.err).Error()
```

Comment:
```suggestion
	return fmt.Sprintf("invalid rpc control message %s validation limit configuration: %s", e.controlMsg, e.err.Error())
```
not sure we need to construct an error just to get the string representation